tests/sqllogictests/src/main.rs

```diff
@@ -198,9 +198,14 @@ async fn run_suits(suits: ReadDir, client_type: ClientType) -> Result<()> {
         return Ok(());
     }
     // Run all tasks parallel
+    let num_of_tasks = tasks.len();
```

Comment:
One task means one test file. The number of tests should be the sum of tests in each test file.

Reply:
Thank for pointing it out, please take another look.